### ISSUE TITLE
Response of the multiplication board to hover should be improved

### ISSUE THREAD

**Issue by jbphet** [phetsims/arithmetic#118]
The multiplication board (or times table, or whatever it is called) shows a hand over a cell when the mouse hovers there, and the color of the cell changes, as does the color of all cells below it on the horizontal and vertical axes.  This behavior appears to be a bit laggy sometimes, and feels somewhat awkward.  I just compared it to the original Flash simulation, and the Flash sim is much more responsive.  This should be improved.

**Comment by jbphet** [comment]
Currently, the cursor is set to 'pointer' for the cells in the multiplication table in the Factor screen.  If I remove this setting, the response time improves dramatically.

**Comment by jbphet** [comment]
This sim is on hold for a while, so I'm unassigning this issue until work resumes.

**Comment by jbphet** [comment]
This issue is only relevant to the "Factor" screen, since the "Multiply" and "Divide" screens don't have the feature where cells highlight on hover.

**Comment by jbphet** [comment]
After some testing, it appears that the grid lines and the "prompt hand" are both pickable, and I think this contributes to feeling of poor performance.

**Comment by jbphet** [comment]
I made some commits on this, but used the wrong ticket number (18 instead of 118).  See 6a4b4d5c700ae6adf2878e8337c3f9d6d72c8d59 and b6b94ffa082970cec260de11bfbe90363987f3d8.  One of these makes the overlay hand non-pickable, the other makes the background use the same cursor as the cells so that the cursor isn't changing on moves between cells.  This seems to have improved responsiveness a lot.  Also, we had an issue a while back on Chrome where cursor changes caused a lot of lag, see https://github.com/phetsims/scenery/issues/275.  This was either fixed or went away due to Chrome changes.  At any rate, the responsiveness is now good.  Closing.
